packages/payment-experience-api/src/lib/vismaPay.ts

```diff
@@ -58,16 +82,20 @@ const canRetryPayment = (p: VismaStatus) => {
   return !paymentPaid && canRetry
 }
 
-const isFailure = (p: VismaStatus) => {
-  const { paymentPaid, canRetry } = p
-  return !paymentPaid && !canRetry
-}
+// const isFailure = (p: VismaStatus) => {
```

Comment:
Do we need to leave this commented code here?

Reply:
Oh good point, I will remove this.